content/en/docs/setup/install/multiple-controlplanes/index.md

```diff
@@ -0,0 +1,299 @@
+    1.17-alpha.f5212a6f7df61fd8156f3585154bed2f003c4117
+    httpbin-9dbd644c7-xzgfm.app-ns-2                      Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED       NOT SENT    istiod-usergroup-2-658d6458f7-slpd9
+    1.17-alpha.f5212a6f7df61fd8156f3585154bed2f003c4117
+    istio-ingressgateway-7bc5b4c97d-bvsvg.usergroup-2     Kubernetes     SYNCED     SYNCED     SYNCED     NOT SENT     NOT SENT    istiod-usergroup-2-658d6458f7-slpd9
+    1.17-alpha.f5212a6f7df61fd8156f3585154bed2f003c4117
+    sleep-78ff5975c6-fthmt.app-ns-2                       Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED       NOT SENT    istiod-usergroup-2-658d6458f7-slpd9
+    1.17-alpha.f5212a6f7df61fd8156f3585154bed2f003c4117
+    sleep-78ff5975c6-nxtth.app-ns-3                       Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED       NOT SENT    istiod-usergroup-2-658d6458f7-slpd9
+    1.17-alpha.f5212a6f7df61fd8156f3585154bed2f003c4117
+    {{< /text >}}
+
+### Verify the application connectivity is ONLY within the respective usergroup
+
+1.  Send a request from the `sleep` pod in `app-ns-1` in `usergroup-1` to the `httpbin` service in `app-ns-2` in `usergroup-2`, the communication should fail:
+
+    {{< text bash >}}
+    $ kubectl -n app-ns-1 exec "$(kubectl -n app-ns-1 get pod -l app=sleep -o jsonpath={.items..metadata.name})" -c sleep -- curl -sIL http://httpbin.app-ns-2.svc.cluster.local:8000
+    HTTP/1.1 503 Service Unavailable
+    content-length: 95
+    content-type: text/plain
+    date: Sat, 24 Dec 2022 06:54:54 GMT
+    server: envoy
+    {{< /text >}}
+
+1.  Send a request from the `sleep` pod in `app-ns-2` in `usergroup-2` to the `httpbin` service in `app-ns-3` in `usergroup-2`, the communication should work:
+
+    {{< text bash >}}
+    $ kubectl -n app-ns-2 exec "$(kubectl -n app-ns-2 get pod -l app=sleep -o jsonpath={.items..metadata.name})" -c sleep -- curl -sIL http://httpbin.app-ns-3.svc.cluster.local:8000
+    HTTP/1.1 200 OK
+    server: envoy
+    date: Thu, 22 Dec 2022 15:01:36 GMT
+    content-type: text/html; charset=utf-8
+    content-length: 9593
+    access-control-allow-origin: *
+    access-control-allow-credentials: true
+    x-envoy-upstream-service-time: 3
+    {{< /text >}}
+
+## Cleanup
+
+1.  Clean up the first usergroup:
+
+    {{< text bash >}}
+    $ istioctl uninstall --revision usergroup-1
+    $ kubectl delete ns app-ns-1 usergroup-1
+    {{< /text >}}
+
+1.  Clean up the second usergroup:
+
+    {{< text bash >}}
+    $ istioctl uninstall --revision usergroup-2
+    $ kubectl delete ns app-ns-2 app-ns-3 usergroup-2
+    {{< /text >}}
+
+{{< warning >}}
+Cluster Administrator has to make sure that the Mesh Administrator DO NOT have the permission to invoke global `istioctl uninstall --purge`
+as that will uninstall all the control planes in the cluster.
```

Comment:
```suggestion
A Cluster Administrator must make sure that Mesh Administrators DO NOT have permission to invoke the global `istioctl uninstall --purge` command,
because that would uninstall all control planes in the cluster.
```